components/cluster/command/root.go

```diff
@@ -49,6 +50,8 @@ var (
 	skipConfirm bool
 )
 
+var tidbSpec *meta.SpecManager
```

Comment:
I think the `tidbSpec` should the global variable of `pkg/cluster/spec` and make it reusable in https://github.com/pingcap/tiup/pull/567/files#diff-fdca3c2d2bcb7afb276475bee9c63c47R51

Reply:
resolve in https://github.com/pingcap/tiup/pull/567/commits/a82ee6c3f48fd751629f13db687915dc646e3c91